rpc/jsonrpc/client/ws_client.go

```diff
@@ -213,6 +213,7 @@ func (c *WSClient) Stop() error {
 	}
 	// only close user-facing channels when we can't write to them
 	c.wg.Wait()
+	c.PingPongLatencyTimer.Stop()
```

Comment:
I don't think it's used anywhere, right? If so, let's just remove it.

Reply:
i think used in latency stats